### PR TITLE
Add missing mime type mappings.

### PR DESCRIPTION
**Pull request by claudiamurialdo**
Issue:92003

### CODE REVIEW

File: dotnet/src/dotnetframework/GxClasses/Core/GXApplication.cs [hunk 1]
```diff
@@ -3047,6 +3049,7 @@ public string ExtensionForContentType(string contentType)
 															 new string[] {"mov"    , "video/quicktime"},
 															 new string[] {"qt"		, "video/quicktime"},
 															 new string[] {"avi"    , "video/x-msvideo"},
+															 new string[] {"m4a"	, "audio/x-m4a"},
```
Comment (mcrispino) [inline dotnet/src/dotnetframework/GxClasses/Core/GXApplication.cs:3052]: Why is it twice? (see line 3029)

Did you mean to add the `m4v` extension?
```suggestion
															 new string[] {"m4v"	, "video/x-m4v"},
```

**Review by mcrispino — CHANGES_REQUESTED** [review]

File: dotnet/src/dotnetframework/GxClasses/Core/GXApplication.cs (hunk 1 above)
Comment (claudiamurialdo) [reply dotnet/src/dotnetframework/GxClasses/Core/GXApplication.cs:3052]: We need the two of them for compatibility right? 
"m4a"<->"audio/mp4"
"m4a"<->"audio/x-m4a"
Remember server-side uses the mapping in both ways.

Comment (claudiamurialdo) [reply dotnet/src/dotnetframework/GxClasses/Core/GXApplication.cs]: Fixed

Comment (mcrispino) [reply dotnet/src/dotnetframework/GxClasses/Core/GXApplication.cs]: Thanks!

**Comment by mcrispino** [comment]
Already merged, but anyway, LGTM now.

**Pull request merged**